src/Symfony/Component/ErrorHandler/Tests/ErrorEnhancer/ClassNotFoundErrorEnhancerTest.php

```diff
@@ -27,13 +27,15 @@ public static function setUpBeforeClass(): void
                 continue;
             }
 
+            $function = $function[0];
+
             // get class loaders wrapped by DebugClassLoader
-            if ($function[0] instanceof DebugClassLoader) {
-                $function = $function[0]->getClassLoader();
```

Comment:
i would debug the actual value here first.. to see if we can understand what happened.

overall the ComposerClassLoader is not a callable/invokable type itself, so i tend to believe we need a 2nd is_array check before accessing `$function[0]` secondly.